src/github/prComment.ts

```diff
@@ -471,3 +482,23 @@ ${lineContents}
 		return new vscode.MarkdownString(this.rawComment.body);
 	}
 }
+
+export function replaceImages(markdownBody: string, htmlBody: string, host: string = 'github.com') {
+	const originalExpression = new RegExp(`https:\/\/${host}\/.+\/assets\/([^\/]+\/)?(?<uuid>${UUID_EXPRESSION.source})`);
+	let originalMatch = markdownBody.match(originalExpression);
+	const htmlHost = host === 'github.com' ? 'githubusercontent.com' : host;
+
+	while (originalMatch) {
+		if (originalMatch.groups?.uuid) {
+			const htmlExpression = new RegExp(`https:\/\/([^"]*${htmlHost})\/[^?]+${originalMatch.groups.uuid}[^"]+`);
```

Comment:
I think `htmlHost` and maybe the `uuid` needs to be escaped for use in the regular expression. Otherwise `.` will match any char instead of a literal `.`

Reply:
Thank you for catching these!